### PR TITLE
tctl: Improve UX for Commands Discoverability

### PR DESCRIPTION
**Pull request by feedmeapples**
- Origin issue: 

# Summary
Improves tctl commands discoverability

# Basic example
Before 
``` bash
tctl workflow activity complete {flags}
```
After 
``` bash
tctl activity complete {flags}
```

# Motivation
Tctl UX made it unnecessary harder to find commands such as activity commands

# Drawbacks
The proposal changes tctl api

Why should we *not* do this? Please consider:

# Alternatives

### CODE REVIEW

**Comment by feedmeapples** [comment]
created a new PR as have removed the old personal fork some time ago

File: cli/0000-cli-improve-ux.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+For start, support the following default flags through the file:
+ - `--namespace`
+
+Add `config` command to open the config in default editor
+
+ Use XDG-spec to locate the config file:
+ ~/.config/tctl on Unix
+ %LOCALAPPDATA%\tctl on Windows
+
+### Standardize wording
```
Comment (feedmeapples) [inline cli/0000-cli-improve-ux.md:73]: haven't re-visited this, tbu

**Review by sw-yx — APPROVED** [review]
we reviewed this this week and i gave Ruslan some comments on what else we could do, but mostly i like the direction of this.

File: cli/0000-cli-improve-ux.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+- Standardize wording
+- Rework the short versions of commands
+- Bring consistent schema for operations on multiple entities
+- Use hyphens instead of underscores
+- Lower priority: Improve help command output/UI
+
+### Non-goals
+
+- Creating any kind of new CLI business logic
+- Refactoring or improving CLI code base
```
Comment (feedmeapples) [inline cli/0000-cli-improve-ux.md:24]:
```suggestion
```

File: cli/0000-cli-improve-ux.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+- Support default configs file
+- Standardize wording
+- Rework the short versions of commands
+- Bring consistent schema for operations on multiple entities
+- Use hyphens instead of underscores
+- Lower priority: Improve help command output/UI
+
+### Non-goals
+
+- Creating any kind of new CLI business logic
```
Comment (feedmeapples) [inline cli/0000-cli-improve-ux.md:23]:
```suggestion
- Creating new CLI business logic
```

**Comment by feedmeapples** [comment]
i'm going to merge the part that was actually already implemented closer to release v1.0.0. Then create an updated PR (also with some changes) with the remaining UX proposal update, let's then send this for community review @sw-yx

File: cli/0000-cli-improve-commands-discoverability.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Before 
+``` bash
+tctl workflow activity complete {flags}
+```
+After 
+``` bash
+tctl activity complete {flags}
+```
+
```
Comment (feedmeapples) [inline cli/0000-cli-improve-commands-discoverability.md:30]:
```suggestion
```

**Pull request merged**